scripts/generate-slug.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Categories of type "generated-index" do not represent the document itself.
+ * By default, their URLs include "category" instead of the document's name,
+ * meaning the document's name does not appear in the URL. This makes it
+ * impossible to show a GitHub link for the repo at the top of the page.
+ *
+ * For example,
+ * - Ignite's Boilerplate: "/ignite-cli/boilerplate/"
+ * - React Native MLKit's Image Labeling: "/category/image-labeling"
+ *
+ * This script gathers all "generated-index" type categories and updates
+ * their entries with the corresponding slug that includes the document name.
+ */
+const fs = require("fs");
+const path = require("path");
+
+const baseDocsDir = path.join(__dirname, "../docs");
+
+function updateCategoryFile(filePath, dirName, subDirName) {
+  const data = JSON.parse(fs.readFileSync(filePath, "utf-8"));
+  if (data.link && data.link.type === "generated-index") {
+    const newSlug = `/${dirName}/${subDirName}/list`;
+    data.link.slug = newSlug;
+    fs.writeFileSync(filePath, JSON.stringify(data, null, 2));
+    console.log(`Updated slug for ${filePath}: ${newSlug}`);
```

Comment:
[nits] let's please drop console.logs

Reply:
My intention was to notify the devs that the file has been manually modified while `yarn install`. But you think that's unnecessary, I'll drop it. :+1: